pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1744,6 +1765,62 @@ def init_arc_param(arctype):
         doc="Treatment construction/expansion capital cost for selected increment [currency/(volume/time)]",
     )
 
+    model.p_tau_TreatmentExpansionLeadTime = Param(
```

Comment:
Can you add the new parameters which have been created to the parameter list in the documentation? I think simply adding them to the list at the top of the page should be sufficient.

Reply:
Done!